applications/app/services/ApplicationsInteractiveRendering.scala

```diff
@@ -18,13 +19,24 @@ object USElection2020AmpPage extends RenderingTier
   1. Implement the support for it in DCR, or
   2. Implement support for it directly in the [applications] app, using the AMP page already present in CAPI.
 
-  The former wold have taken too long so we went for the latter.
+  The former would have taken too long so we went for the latter.
+
+  ----------------
+  Author: Pascal
+  Date: 21st April 2021
+
+  We are now moving towards supporting interactives in DCR 🙂
  */
 
 object ApplicationsInteractiveRendering {
   def getRenderingTier(path: String)(implicit request: RequestHeader): RenderingTier = {
     val isSpecialElection = ApplicationsUSElection2020AmpPages.pathIsSpecialHanding(path)
     val isAmp = request.host.contains("amp")
-    if (isSpecialElection && isAmp) USElection2020AmpPage else Regular
+    val forceDCR = request.forceDCR
+    (isSpecialElection, isAmp, forceDCR) match {
```

Comment:
I wonder if this might just be simpler as:

```
if (isSpecialElection && isAmp) USElection2020AmpPage
else if (forceDCR) DotcomRendering
else Regular
```

Not a big deal either way, but matching on three booleans is a bit confusing perhaps.

Reply:
Oh Yeah. Happy to use the `if`style :)